pkg/broker/rbac.go

```diff
@@ -66,6 +66,11 @@ func NewBrokerAdminRole() *rbacv1.Role {
 				APIGroups: []string{"rbac.authorization.k8s.io"},
 				Resources: []string{"rolebindings"},
 			},
+			rbacv1.PolicyRule{
```

Comment:
This role is for "subctl" only, if you want the clusters to be able to do this, add it to the NewBrokerClusterRole only

Reply:
Without this admin was not able to grant cluster roles and deployment was failing hence added this one.

Reply:
But that's unrelated, I will investigate this afterwards. I added this: https://github.com/submariner-io/submariner-operator/issues/326